lib/broccoli/ember-app.js

```diff
@@ -637,7 +637,7 @@ EmberApp.prototype._addonTree = function _addonTree() {
   });
 
   var addonES6 = new Funnel(addonTrees, {
-    srcDir: 'modules',
+    include: [new RegExp('^modules.*\\.js$')],
```

Comment:
I tried:

```js
srcDir: 'modules',
include: [new RegExp('.*\\.js$')],
```

but it gives an error in broccoli-funnel. So I used the modules in the regex. Does it behaves in the same way?